Makefile

```diff
@@ -1,145 +1,224 @@
-# Parameters
-PROJECT	= frontend-ssr-template
-K6 = $(DOCKER) run -v ./src/test/load:/loadTests --net=host --rm k6 run --summary-trend-stats="avg,min,med,max,p(95),p(99)"
-K6_BIN = ./k6
-
-# Executables: local only
-PNPM_BIN		= pnpm
-DOCKER			= docker
-DOCKER_COMPOSE	= docker compose
-MAKE 			= make
-
-# Executables
-EXEC_NODEJS	= $(DOCKER_COMPOSE) exec prod
-PNPM      	= $(EXEC_NODEJS) pnpm
-PNPM_RUN    = $(PNPM) run
-GIT         = git
+include .env
+export
+
+DOCKER_COMPOSE              = docker compose
+
```

Comment:
Please remove unnecessary comments and be consistent with the amount of new lines between sections, sometimes there is one new line and sometimes there is two lines between sections